field_friend/interface/components/leaflet_map.py

```diff
@@ -55,12 +56,10 @@ def __init__(self, system: 'System', draw_tools: bool) -> None:
         self.robot_marker: Marker | None = None
         self.drawn_marker = None
         self.row_layers: list = []
-        self.active_field: str | None = None
-        self.set_active_field()
         self.update_layers()
         self.zoom_to_robot()
-        self.field_provider.FIELDS_CHANGED.register(self.set_active_field)
         self.field_provider.FIELDS_CHANGED.register(self.update_layers)
+        self.field_provider.FIELD_SELECTED.register(self.update_layers)
```

Comment:
```suggestion
        self.field_provider.FIELDS_CHANGED.register_ui(self.update_layers)
        self.field_provider.FIELD_SELECTED.register_ui(self.update_layers)
```
Do we need `register_ui` here as well?